client/view/Form/Form.js

```diff
@@ -83,7 +71,7 @@ function renderRegionSelectOptions() {
     let renderOption = (id, name) => {
       let option = document.createElement('option')
       option.value = id
```

Comment:
Let's use the new `createTag` utility for better readibility

```suggestion
      let option = createTag('option', [], name)
      option.value = id
```